### ISSUE TITLE
Issues with Bank Reconciliation 

### ISSUE THREAD

**Issue by Basawa-SELCO** [frappe/erpnext#15496]
Scenario:
1) Multiple Debits posted against multiple Accounts from a single Credit Account in a Single Journal Entry.
2) In the Bank Reconciliation Table, There are separate rows for Debits and Credit Accounts for the same transaction and not displayed as a Debit - Credit Account pair. As in, the table has only "Against Account"and no "For Account"
3) Also, the "Ägainst Account" displays all the Debit Accounts for each of the entries making it difficult to identify.
3) This makes it difficult to track the Entries against the Credit Account.

How this can be handled?

**Comment by nikkothari22** [comment]
Most of these are fixed via the Bank Entry feature in Mint. Assigning it to me so that we can close it once Mint is merged into ERPNext
